{"name": "test_member", "status": "broken", "statusDetails": {"message": "json.decoder.JSONDecodeError: Expecting ':' delimiter: line 11 column 28 (char 358)", "trace": "@pytest.fixture(scope=\"class\")\n    def uiTestData():\n        # get env name\n        # env_name = os.getenv(\"environment\")\n        env_name = \"dev\"\n        print(\"env_name : \"+env_name)\n        # read json for access the env data\n        with open(f\"{os.getcwd()}/ConfigFiles/{env_name}/UI/UiTestData.json\", 'r') as js:\n>           testData = json.load(js)\n\nTestCases\\conftest.py:150: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\json\\__init__.py:293: in load\n    return loads(fp.read(),\n..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\json\\__init__.py:346: in loads\n    return _default_decoder.decode(s)\n..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\json\\decoder.py:337: in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <json.decoder.JSONDecoder object at 0x000001F161CE8210>\ns = '{\\n    \"url_11\" : \"https://amdoq11.medecision.com/ACM/medSupportAltLogin.faces\",\\n    \"username\" : \"sbeaston\",\\n    \"...mo03\"\\n      },\\n  \\n    \"facility\":{\\n      \"facility_name\": \"Teamfacility\",\\n      \"facility_id\": \"FA03\"\\n    }\\n  }'\nidx = 0\n\n    def raw_decode(self, s, idx=0):\n        \"\"\"Decode a JSON document from ``s`` (a ``str`` beginning with\n        a JSON document) and return a 2-tuple of the Python\n        representation and the index in ``s`` where the document ended.\n    \n        This can be used to decode a JSON document from a string that may\n        have extraneous data at the end.\n    \n        \"\"\"\n        try:\n>           obj, end = self.scan_once(s, idx)\nE           json.decoder.JSONDecodeError: Expecting ':' delimiter: line 11 column 28 (char 358)\n\n..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\json\\decoder.py:353: JSONDecodeError"}, "attachments": [{"name": "stdout", "source": "d0d2c549-0bef-46e8-bc65-31358a80d1a3-attachment.txt", "type": "text/plain"}], "start": 1674197712919, "stop": 1674197712919, "uuid": "2a6c9097-b76f-4ad6-b811-61f0ac679201", "historyId": "fcbd87c39b5178d291b12e7bf1e410ab", "testCaseId": "9cdac9e9efb952e294eadcfe8cce2eec", "fullName": "TestCases.testsuites.test_sample.TestSample1#test_member", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('test_setup')"}, {"name": "parentSuite", "value": "TestCases.testsuites"}, {"name": "suite", "value": "test_sample"}, {"name": "subSuite", "value": "TestSample1"}, {"name": "host", "value": "WSAMZN-7K50OVEF"}, {"name": "thread", "value": "9548-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.testsuites.test_sample"}]}